pkg/overlay/service_test.go

```diff
@@ -0,0 +1,44 @@
+package overlay
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"google.golang.org/grpc"
+	proto "storj.io/storj/protos/overlay" // naming proto to avoid confusion with this package
+)
+
+const (
+	port = 50501
+)
+
+func TestNewServer(t *testing.T) {
+	lis, err := net.Listen("tcp", fmt.Sprintf(":%d", port))
```

Comment:
for tests, you should probably listen on "127.0.0.1:0". you probably know this but using 127.0.0.1 means it will bind to the loopback interface only, and using port 0 means it will get any available free port. you can get the port it chose by inspecting lis after it's returned.

Reply:
otherwise, these tests can't run in parallel

Reply:
cool! I actually didn't know that!